### PR TITLE
reading to and writing from file implemented, report needs review

### PR DESCRIPTION
**Pull request by TymurProkhorov**
Can`t realise how to separate different logical parts to different methods, so now I`ve just marked it by comments.

### CODE REVIEW

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 1]
```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.*;
```
Comment (ChumakAn) [inline src/main/java/core/basesyntax/WorkWithFile.java:3]: common mistake 
image

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 2]
```diff
@@ -1,8 +1,59 @@
...
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        // read from file
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```
Comment (ChumakAn) [inline src/main/java/core/basesyntax/WorkWithFile.java:11]: what about try-with-resources?

**Review by ChumakAn — CHANGES_REQUESTED** [review]
try to make separate methods with this code, you can divide it in methods with the similar parts as you've added comments 
Also remember about reading the common mistakes and avoiding them. Also you have some magic numbers and strings, make them contants

File: src/main/java/core/basesyntax/WorkWithFile.java (hunk 1 above)
Comment (TymurProkhorov) [reply src/main/java/core/basesyntax/WorkWithFile.java:3]: I saw it, but couldnt fix. Thanks for help)

**Review by ChumakAn — CHANGES_REQUESTED** [review]
Your code didn't pass the tests, you can see the detailed logs in the Java CI build failure report. When you call createReport method, you don't save the result of report to any variable and don't pass it to the writeToFile method, as a result, you have nothing to write into the file

**Review by ChumakAn — APPROVED** [review]
LGTM

**Pull request closed without merging**